src/compose/ComposeBox.js

```diff
@@ -320,14 +322,18 @@ class ComposeBox extends PureComponent<Props, State> {
   };
 
   handleSend = () => {
-    const { dispatch, narrow, caughtUp, _ } = this.props;
+    const { dispatch, narrow, caughtUp, _, isTopicMandatory } = this.props;
     const { message } = this.state;
+    const topic = this.state.topic;
```

Comment:
nit: combine with previous line instead of making a new line, like

```js
const { message, topic } = this.state;
```

Reply:
Make sense, done!